lib/cadet_web/views/chat_view.ex

```diff
@@ -1,8 +1,12 @@
 defmodule CadetWeb.ChatView do
   use CadetWeb, :view
 
-  def render("conversation_init.json", %{conversation_id: id, last_message: last}) do
-    %{conversationId: id, response: last}
+  def render("conversation_init.json", %{
+        conversation_id: id,
+        last_message: last,
+        max_content_size: size
```

Comment:
I'm thinking if there is a need to configure it this way (as opposed to hardcoding), but I am fine with this as well. What do you think?